# HG changeset patch
# User Dan Villiom Podlaski Christiansen <dan@cabo.dk>
# Date 1393341609 -3600
#      Tue Feb 25 16:20:09 2014 +0100
# Node ID d6c1eff53c2a31be94715ff6ba17aac16a511a32
# Parent  b25ba495c84b27bbb32382e3beb4ddc37c5943c5
Add missing tab index attributes.

diff --git a/js/tinymce/classes/ui/ColorButton.js b/js/tinymce/classes/ui/ColorButton.js
--- a/js/tinymce/classes/ui/ColorButton.js
+++ b/js/tinymce/classes/ui/ColorButton.js
@@ -66,7 +66,7 @@ define("tinymce/ui/ColorButton", [
 			var image = self.settings.image ? ' style="background-image: url(\'' + self.settings.image + '\')"' : '';
 
 			return (
-				'<div id="' + id + '" class="' + self.classes() + '">' +
+				'<div id="' + id + '" class="' + self.classes() + '" tabindex="-1">' +
 					'<button role="presentation" hidefocus type="button" tabindex="-1">' +
 						(icon ? '<i class="' + icon + '"' + image + '></i>' : '') +
 						'<span id="' + id + '-preview" class="' + prefix + 'preview"></span>' +
diff --git a/js/tinymce/classes/ui/SplitButton.js b/js/tinymce/classes/ui/SplitButton.js
--- a/js/tinymce/classes/ui/SplitButton.js
+++ b/js/tinymce/classes/ui/SplitButton.js
@@ -72,7 +72,7 @@ define("tinymce/ui/SplitButton", [
 			var icon = self.settings.icon ? prefix + 'ico ' + prefix + 'i-' + self.settings.icon : '';
 
 			return (
-				'<div id="' + id + '" class="' + self.classes() + '">' +
+				'<div id="' + id + '" class="' + self.classes() + '" tabindex="-1">' +
 					'<button type="button" hidefocus tabindex="-1">' +
 						(icon ? '<i class="' + icon + '"></i>' : '') +
 						(self._text ? (icon ? ' ' : '') + self._text : '') +
@@ -116,4 +116,4 @@ define("tinymce/ui/SplitButton", [
 			return self._super();
 		}
 	});
-});
\ No newline at end of file
+});
